components/IscnFormHeader.vue

```diff
@@ -0,0 +1,34 @@
+<template>
+  <div class="flex flex-row items-start justify-between">
+    <Label
+      class="w-min"
+      :text="$t('UploadForm.title.registerISCN')"
+      tag="div"
+      preset="p5"
+      valign="middle"
+      content-class="font-semibold whitespace-nowrap text-like-green"
+      prepend-class="text-like-green"
+    >
+      <template #prepend>
+        <IconRegister />
+      </template>
+    </Label>
+    <div class="flex flex-col items-end">
+      <Stepper :step="step" />
+      <Label
+        preset="p6"
+        :text="`${$t('Registration.step')} ${step}/4`"
```

Comment:
Use interpolation is better
```suggestion
        :text="$t('Registration.step', { step, total: 4 })"
```